.travis.yml

```diff
@@ -1,17 +1,36 @@
-language: go
+matrix:
+  include:
+    - language: go
+      name: test make process
+      go:
+        - 1.13.x
+      sudo: false
+      cache:
+        directories:
+          - /home/travis/.cache/go-build
+          - /home/travis/gopath/pkg/mod
+      env:
+        - GOPROXY=https://proxy.golang.org/
+      script:
+        - make check
 
-go:
-  - 1.13.x
-
-sudo: false
-
-cache:
-  directories:
-  - /home/travis/.cache/go-build
-  - /home/travis/gopath/pkg/mod
-
-env:
-  - GOPROXY=https://proxy.golang.org/
-
-script:
-  - make check
+    - language: ruby
+      rvm: 2.6
+      name: Site Proofing
+      sudo: false # route your build to the container-based infrastructure for a faster build
+      cache: bundler
+      env:
+        - NOKOGIRI_USE_SYSTEM_LIBRARIES=true # speeds up installation of html-proofer
+        - SEARCH_DIRECTORIES="site"
+      addons:
+        apt:
+          packages:
+            - libcurl4-openssl-dev
+      before_install:
+        - chmod +x ./hack/travis/travis-check-files-changed.sh
```

Comment:
You should not need to chmod these. Git preserves permissions, so chmod the script and commit them and they will still be executable.